### DISCUSSION TITLE
Fresh Ubuntu 21.10, ran pf-elk script, but no dashboard found, or data

### DISCUSSION THREAD

**Discussion by benisai** [pfelk/pfelk#375] [Q&A] [answered]
I spun up a fresh ubuntu and ran the PF-elk script, but I see no dashboards. 
I ran the template and dashboard scripts and then I do see dashboards under Kibana. so thats good. 
I pointed my pfsense syslog to my pf-elk server, HOWEVER, I see no data on the dashboard. 

I'm new to ELK and dont really know where to look in the UI, so much to see in elastic. 

it took a bit but I see data under firewall.  but my device is not opnsense, its pfsense, can this be changed?

**Comment by a3ilson** [comment, marked as answer]
@benisai 

Update Observer Fields
* to amend the observer fields, edit the 02-types.conf file. Update the applicable values and restart logstash 

* Issue #373 appears to echo the same issue (i.e. templates failing to install via script). I am currently testing/updating the output to datastreams and also noted that the Kibana saved objects failed to import with v7.16.X and suspect it to be a break change. New Kibana objects will be built momentarily and further testing. However, reverting to the last known working version will resolve this until the issue can be corrected.

[version 20.10](https://github.com/pfelk/pfelk/releases/tag/20.10)
